feat(#297): New rule: Prefer Unquoted Atoms (#355)
inaka/elvis_core: doc_rules/elvis_style/prefer_unquoted_atoms.md

> # Prefer unquoted atoms

Reviewer: This file should be moved to `elvis_text_style` folder.

> Do not use quotes on atoms that are not need it.

Reviewer: ```suggestion
Do not use quotes on atoms that don't need to be quoted.
```

Author: Sorry for these mistakes.
